modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+You must specify at least 1 conflict log collection, which is the default.
++
+When you do not specify any conflict logging rules, all conflicts detected during the replication are logged into the default conflict log collection.
+The conflict logging rules allow granular configuration of conflict log collections for specific scopes and collections in the replication.
+You can override the default log collection by either specifying which scopes and collections must have their conflicts logged, or assigning different conflict log collections for each.
++
+For detailed information about how to enable and configure the conflict logging feature while creating or updating a replication, see the following:
+
+** xref:manage:manage-xdcr/create-xdcr-replication.adoc#xdcr-ui-settings-for-conflict-logging
+[Replication Settings for XDCR Conflict Logging], to set from UI.
+
+** xref:rest-api:rest-xdcr-create-replication.adoc[Creating a Replication] by xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Enabling and Configuring Conflict Logging], to set through REST API.
+
+. When XDCR logs a conflict, it creates 3 related documents in the conflict log collection specified by you.
+The 3 documents together is referred to as the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record].
+As the conflict log collections are regular collections, you can programmatically access and review the documents.
+For examples about different ways to query or process the conflict log information, see xref:learn:clusters-and-availability/xdcr-viewing-conflict-logs.adoc[Viewing Conflict Logs].
+
+[#conflict-collection]
+== Conflict Log Collection
+
+A conflict log collection, or conflict collection, is the collection you specify to store conflict event messages and conflicting documents when you configure conflict logging for XDCR replication.
+Each conflict event creates 3 documents in the conflict log collection; 1 conflict event message document and 2 conflicting documents.
+Together, these 3 documents form a xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record].
+
+For information about configuring a collection to log conflicts, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Enabling and Configuring Conflict Logging].
+
+You can specify a regular collection as a conflict log collection in the cluster, where you're creating the replication.
+You can also name the source collection, which is a part of the replication process, as a conflict log collection.
+The conflict record documents are not replicated.
+
+After the XDCR conflict logger creates the conflict log collection, you can manage the conflict record documents in them.
```

Comment:
Please don't say "XDCR ... creates the conflict log collection" -- this will confuse users into thinking that XDCR creates collections.  XDCR never creates collections -- the collections must already exists -- i.e. the users must have created the collections.  Also, "you can manage" makes it sound like they could, if they want ... otherwise, somebody else will do it... which is not true.

You must manage the conflict record documents after XDCR creates them in the conflict collection, as XDCR will not touch them again.